### ISSUE TITLE
Add standard daily weather statistics

### ISSUE THREAD

**Issue by JoshuaCarroll** [JoshuaCarroll/Raspberry-Pi-Weather-Station-Dashboard#12]
Add additional standard weather statistics such as:
- Rainfall amount since midnight
- Date/time of the start of the most recent storm
- Date/time of the end of the most recent storm
- Total rainfall of the most recent storm
- High temp since midnight
- Low temp since midnight

**Comment by JoshuaCarroll** [comment]
Create a new stored procedure and include it in the setup instructions (best if most SP's are created via a single script).

```
SELECT @RainfallSinceMidnight:=SUM(RAINFALL) FROM WEATHER_MEASUREMENT WHERE created >= DATE(NOW());

SELECT @stormEnd:=created FROM WEATHER_MEASUREMENT WHERE rainfall > 0 ORDER BY created DESC LIMIT 1;
SELECT @stormStart:=created FROM WEATHER_MEASUREMENT WHERE rainfall = 0 AND created < @stormEnd ORDER BY created DESC LIMIT 1;
SELECT @stormTotal:=SUM(Rainfall) FROM WEATHER_MEASUREMENT WHERE created >= @stormStart AND created <= @stormEnd;

SELECT @LowSinceMidnight:= AMBIENT_TEMPERATURE FROM WEATHER_MEASUREMENT WHERE created >= DATE(NOW()) ORDER BY AMBIENT_TEMPERATURE ASC LIMIT 1;
SELECT @HighSinceMidnight:= AMBIENT_TEMPERATURE FROM WEATHER_MEASUREMENT WHERE created >= DATE(NOW()) ORDER BY AMBIENT_TEMPERATURE DESC LIMIT 1;

SELECT 
    @RainFallSinceMidnight AS RainFallSinceMidnight,
    @stormStart AS LastStormStart,
    @stormEnd AS LastStormEnd,
    @stormTotal AS LastStormTotal,
    @LowSinceMidnight AS LowSinceMidnight,
    @HighSinceMidnight AS HighSinceMidnight;
```

**Comment by JoshuaCarroll** [comment]
Fixed by commit 1041c14.  Will be included in the next merge to master.
